var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Babelstream(CMakePackage):
+    """Measure memory transfer rates to/from global device memory on GPUs.
+    This benchmark is similar in spirit, and based on, the STREAM benchmark for CPUs."""
+
+    homepage = "https://github.com/UoB-HPC/BabelStream"
+    git = "https://github.com/UoB-HPC/BabelStream.git"
+    url = "https://github.com/UoB-HPC/BabelStream/archive/refs/tags/v4.0.tar.gz"
+
+    maintainers = ["rob64j", "tomdeakin", "tom91136"]
```

Comment:
One username seems wrong:
```suggestion
    maintainers = ["robj0nes ", "tomdeakin", "tom91136"]
```